doc/development/deprecations.rst

```diff
@@ -9,17 +9,24 @@ deprecations are listed below.
 Pending deprecations
 --------------------
 
-* The :attr:`pennylane.operation.Operator.is_hermitian` property has been deprecated and renamed 
-  to :attr:`pennylane.operation.Operator.is_verified_hermitian` as it better reflects the functionality of this property. 
-  The deprecated access through ``is_hermitian`` will be removed in PennyLane v0.45. 
-  Alternatively, consider using the :func:`pennylane.is_hermitian` function instead as it provides a more reliable check for hermiticity. 
+* The :attr:`pennylane.operation.Operator.is_hermitian` property has been deprecated and renamed
+  to :attr:`pennylane.operation.Operator.is_verified_hermitian` as it better reflects the functionality of this property.
+  The deprecated access through ``is_hermitian`` will be removed in PennyLane v0.45.
+  Alternatively, consider using the :func:`pennylane.is_hermitian` function instead as it provides a more reliable check for hermiticity.
   Please be aware that it comes with a higher computational cost.
-  
+
   - Deprecated in v0.44
   - Will be removed in v0.45
-  
-* Access to the following functions and classes from the ``pennylane.resources`` module are 
-  deprecated. Instead, these functions must be imported from the ``pennylane.estimator`` module.
+
+* Passing a function to the ``gate_set`` argument in the :func:`~pennylane.transforms.decompose` transform
+  is deprecated. The ``gate_set`` argument expects a static iterable of operator type and/or operator names,
+  and the function should be passed to the ``stopping_condition`` argument instead.
+
```

Comment:
Could you include a brief example illustrating this change here and in the corresponding place in the changelog?

Potentially this example from the `qml.transforms.decompose` docstring doc could be adapter here:
```
@partial(qml.transforms.decompose, gate_set=lambda op: len(op.wires)<=2)
@qml.qnode(qml.device("default.qubit"))
def circuit():
    qml.Hadamard(wires=[0])
    qml.Toffoli(wires=[0,1,2])
    return qml.expval(qml.Z(0))```